KTL.js

```diff
@@ -8919,6 +8942,18 @@ function Ktl($, appInfo) {
                 })
 
             },
+            stickTableHeaders: function (viewId, data, numOfRecords, viewHeight) {
+                if (data.length < numOfRecords) return;
+                $(`#${viewId} table, #${viewId} .kn-table-wrapper`)
+                    .addClass('ktlStickyHeader')
```

Comment:
Class "ktlStickyColumn", why is it there? How is it used?

Reply:
Yes I agree I'll remove